### PR TITLE
Support QA automation on Select

### PR DESCRIPTION
**Pull request by dorokhovich-mikhail**

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @pnzau, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Select/Select.component.tsx [hunk 1]
```diff
@@ -370,6 +374,20 @@ export class CustomSelect extends React.Component<Props> {
...
+        return (
+          <SIdWrapper id={`${id}-${key}`}>
+            <Component {...rest}>{children}</Component>
+          </SIdWrapper>
+        );
+      };
+      return res;
+    }, {});
+
+    console.log(components, propsComponents, _components);
```
Comment (pnzau) [inline src/Select/Select.component.tsx:389]: unwanted logs to console in PR.

Also ran the app and the following features break:
the placeholder Select... is being cut out 
also the props breaks the select with checkbox feature

**Comment by hamholla** [comment]
https://dev.azure.com/tkxs/TKXS%20PowerSuite/_workitems/edit/4810/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+### Bug Fixes
+
+* **button:** fixed shadows and border outline in Button ([4b84aac](https://github.com/technekes/cast-ui/commit/4b84aac2279b66b2b4628d2f14b8a512e18a7915))
+* **ci:** trigger build ([#786](https://github.com/technekes/cast-ui/issues/786)) ([4bdcea1](https://github.com/technekes/cast-ui/commit/4bdcea1078199c7988f98dbc15ff7ed19e7a29f6))
+* **component:** tabs border fix ([f5c31f0](https://github.com/technekes/cast-ui/commit/f5c31f0c8841ca3aa9176200613ce5b8995f11bb))
+* **input:** typing on input jumps to last character fix ([4154235](https://github.com/technekes/cast-ui/commit/41542358c8edbf546c45959bdc38b6e12eb01270))
+* **input.component:** removes margin on button ([5559c36](https://github.com/technekes/cast-ui/commit/5559c36eefb561a97b9dfab4e46ed41cdb27afdd))
+* **panelheader:** rotate panel header chevron ([7963884](https://github.com/technekes/cast-ui/commit/7963884e013adb8e443493645c19723de42cab93))
+* **select checkbox:** fixed maximum menu height for select with checkbox ([#784](https://github.com/technekes/cast-ui/issues/784)) ([1683b5c](https://github.com/technekes/cast-ui/commit/1683b5c1b5338faa3d2f6f32772a457db5311b57))
+
```
Comment (pnzau) [inline CHANGELOG.md:13]: Is this caused by the version bump in package.json as well?

**Comment by hamholla** [comment]
Closing this pr. Looks like it was initially branched off of master and causing some issues with commitizen

**Pull request closed without merging**